build/gulp/test_timezones_data.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+const gulp = require('gulp');
+const through = require('through2');
+const remoteSrc = require('gulp-remote-src');
+let tzData = [];
+
+try {
+    tzData = require('../../artifacts/transpiled/ui/scheduler/timezones/ui.scheduler.timezones_data').zones; // eslint-disable-line node/no-missing-require
+} catch(e) {
+    if(e instanceof Error && e.code === 'MODULE_NOT_FOUND') {
+        console.log('Can not load ui.scheduler.timezones_data');
+    } else {
+        throw e;
+    }
+}
+
+gulp.task('test-timezones-data', () => {
+    const momentTimezonesRawUrl = 'https://raw.githubusercontent.com/moment/moment-timezone/develop/data/unpacked/';
+
+    return remoteSrc(['latest.json'], {
+        base: momentTimezonesRawUrl
+    }).pipe(
+        through.obj((file, enc, cb) => {
+            const rawJSON = file.contents.toString();
+            const parsed = JSON.parse(rawJSON);
+            checkTimeZonesParsing(parsed);
+            cb();
```

Comment:
What is cb? )

Reply:
@AntonSermyazhko cb is callBack. Should be called to avoid 'The following tasks did not complete' error. I will rename it)